Tools/Replay/DataFlashFileReader.cpp

```diff
@@ -15,7 +15,9 @@ uint64_t now() {
 }
 
 DataFlashFileReader::DataFlashFileReader() :
-    start_micros(now())
+    fd(-1),
```

Comment:
Why? This is already initialized in the declaration.

Reply:
Can't be too careful.

Removed it and moved initialisation of packets counts into declaration.